batch/batch/front_end/front_end.py

```diff
@@ -1185,7 +1185,8 @@ async def _create_jobs(
             memory_to_worker_types = memory_to_worker_type(cloud)
             if req_memory in memory_to_worker_types:
                 worker_type = memory_to_worker_types[req_memory]
-                req_memory_bytes = cores_mcpu_to_memory_bytes(cloud, req_cores_mcpu, worker_type)
+                machine_type_made = 'n1-'+worker_type+'-'+resources['req_cpu']
```

Comment:
Let's add a utility function named something like "shared pool machine type" that takes the worker type and returns the machine type for the pool of that worker type.